README.md

```diff
@@ -508,21 +536,28 @@ start_termux_server() {
 
 audio_server() {
     select_distro || return 1
+    unmount_chroot
     echo -e "\nRunning Audio Server for $selected_distro..."
     start_termux_server
-    su -c "chroot-distro command $selected_distro \"export DISPLAY=:0 PULSE_SERVER=tcp:127.0.0.1:4713 && dbus-launch --exit-with-session && clear && mpd\""
+    su -c "chroot-distro command $selected_distro \"export DISPLAY=:0 PULSE_SERVER=tcp:127.0.0.1:4713 && dbus-launch --exit-with-session && mpd\""
+    clear
 }
 
 login_cli() {
     select_distro || return 1
+    unmount_chroot
+    clear
     echo -e "\nLogging in as CLI to $selected_distro..."
+    tput cnorm
     read -rp "Enter username: " username
     sudo chroot-distro command "$selected_distro" "su -l $username"
```

Comment:
Are you aware that not all rootfs contains `su` binary by default? If you need to log in with specific user then we should extend `chroot-distro login` command to accept username so that if `su` is available then it will be used to log in, but if it is not then either proper error message is issued, or alternatively things are properly initialized with shell.

Reply:
I would rather see that login command is extended to support non-root users, as then the implementation will be consistent. It is very easy to leak host environment inside the chroot, and either cause undefined behaviour, or cause a security problem.